snowflake-api/src/lib.rs

```diff
@@ -75,6 +80,9 @@ pub enum SnowflakeApiError {
 
     #[error("Unexpected API response")]
     UnexpectedResponse,
+
+    #[error("Missing feature: {0}")]
+    MissingFeature(String),
```

Comment:
failing runtime because of the compile flag is not a good idea

Reply:
This is a tricky one because we can't wrap the whole function in a cfg because it depends the value of the regex. To keep things simple, should we just not have a separate file feature for now?